strr-web/components/bcros/form-section/review/Form.vue

```diff
@@ -55,12 +55,15 @@
               />
               <BcrosFormSectionReviewItem
                 :title="tReview('isUnitOnPrincipalResidenceProperty')"
-                :content="tReview(formState.propertyDetails.isUnitOnPrincipalResidenceProperty)|| '-'"
+                :content="
+                  tReview(formState.propertyDetails.isUnitOnPrincipalResidenceProperty ? 'true' : 'false') || '-'
```

Comment:
@avni-work actually I realized that the '-' will never be shown, because it will always default to 'false' if `isUnitOnPrincipalResidenceProperty` is null or doesn't exists.  Maybe you can double check. Thanks!